FizzBuzz/logic.rb

```diff
@@ -0,0 +1,13 @@
+class Fizz_Buzz
```

Comment:
underscores in class names in ruby?
<img width="687" alt="obraz" src="https://user-images.githubusercontent.com/5008759/61081499-a62a9880-a427-11e9-9476-3690c692524b.png">

please change it to camel case :)

Reply:
also - in general we should stick to best practices of naming the file after the class the file defines. So, here we're aiming more for `FizzBuzz/fizz_buzz.rb` and `class FizzBuzz`